packages/commons/src/file-manager/fileManager.ts

```diff
@@ -46,6 +50,12 @@ export function initFileManager(
       : undefined,
     forcePathStyle: config.s3CustomServer,
     logger: config.logLevel === "debug" ? console : undefined,
+    credentials: config.s3CustomServer
+      ? {
+          accessKeyId: config.minioAccessKey,
+          secretAccessKey: config.minioSecretKey,
+        }
+      : undefined,
```

Comment:
I decided to use custom credentials here because for `bff` we need both the real AWS credentials for `KMS` and the mock credentials for `MINIO`. If I don't override here I cannot differentiate between the real aws profile and the mocked one. 
I agree this is not the optimal solution but I couldn't find a better solution. 
Do you have something specific in mind?

Another approach could be to create a KMS local alternative but apart from the one offered by [localstack](https://www.localstack.cloud/) I couldn't find a solid solution

Reply:
so is real KMS actually used during the tests?

Reply:
Not during tests but during local development